SQL/0000-00-05-ElectrophysiologyTables.sql

```diff
@@ -484,12 +484,14 @@ CREATE TABLE `physiological_annotation_rel` (
 -- Create EEG upload table
 CREATE TABLE `electrophysiology_uploader` (
     `UploadID` int(10) unsigned NOT NULL AUTO_INCREMENT,
-    `UploadedBy` varchar(255) NOT NULL DEFAULT '',
-    `UploadDate` DateTime DEFAULT NULL,
-    `UploadLocation` varchar(255) NOT NULL DEFAULT '',
-    `Status` enum('Not Started', 'In Progress', 'Complete', 'Failed') DEFAULT 'Not Started',
-    `SessionID` int(10) unsigned NOT NULL default '0',
-    `MetaData` TEXT default NULL,
+    `UploadedBy` varchar(255) NOT NULL,
+    `UploadDate` DateTime NOT NULL,
+    `UploadLocation` varchar(255) NOT NULL,
+    `Status` enum('Not Started', 'Decompressed', 'In Progress', 'Complete', 'Failed') DEFAULT 'Not Started',
+    `SessionID` int(10) unsigned,
+    `Checksum` varchar(40) DEFAULT NULL,
+    `AssemblyLocation` varchar(255) DEFAULT NULL,
```

Comment:
It's a schema change I did on HBCD that I propagate on core as well, to set default values that makes more sense. Do we need a patch since the module/schema did not exist before?

Reply:
@laemtl wasn't there a patch created for the new module? If so, maybe that patch needs to be updated? So that existing projects can benefit from the new module. Does that make sense?

Reply:
Oh I just got it, totally make sense. I'm on it.

Reply:
@cmadjar Done